discretisedfield/mesh.py

```diff
@@ -2248,7 +2248,7 @@ def fftn(self, rfft=False):
                 # Shift the region boundaries to get the correct coordinates of
                 # mesh cells.
                 # This effectively does the same as using fftshift
-                dfreq = (freqs[1] - freqs[0]) / 2
+                dfreq = abs(freqs[1] - freqs[0]) / 2
```

Comment:
This change seems reasonable. Why did we not have (need) it before?

Reply:
There was an assumption that `freqs` is always increasing however if we do an fft on a small mesh I found this wasn't the case.